### PR TITLE
README: Fix markdown.

### PR DESCRIPTION
**Pull request by zhegao9**
1. Fix markdown syntax
2. Footnote with `sup`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,10 +139,10 @@
...
-#1. 摘自基于陆兴华原文的《为什么工作有待发明？》，作者：serious：
+[1]. 摘自基于陆兴华原文的《为什么工作有待发明？》，作者：serious：
 [https://www.douban.com/note/761861211/](https://www.douban.com/note/761861211/)
 
-#2. 受启发于forem团队的wen z，感谢askender的修改建议：
+[2]. 受启发于forem团队的wen z，感谢askender的修改建议：
 [https://www.forem.com/future/](https://www.forem.com/future/)
 
-#. 以上文档为09/18/2021的版本，最近版本位于 [https://github.com/diylove/wiki/](https://github.com/diylove/wiki/)
+#. 以上文档为09/18/2021的版本，最新版本位于 [https://github.com/diylove/wiki/](https://github.com/diylove/wiki/)
```
Comment (tiantiandas) [inline README.md:148]: I think this line could be removed.

File: README.md [hunk 2]
```diff
@@ -8,11 +8,11 @@
 阿伦特《人的条件》说：
 “现代人缺失的正是劳动的公共性。”
 在公共性里，我才生产我自己，因而生产我要的社会。
-没得到这样劳动经验的人，是奴隶。#1
+没得到这样劳动经验的人，是奴隶。<sup>[1]</sup>
```
Comment (tiantiandas) [inline README.md:11]: Symbol "#" is always related to an issue.

File: README.md (hunk 1 above)
Comment (2jojo) [reply README.md:148]: ah right!

File: README.md (hunk 2 above)
Comment (2jojo) [reply README.md:11]: good call

File: README.md (hunk 1 above)
Comment (7VLY6hpcgWXfsyF) [reply README.md:148]: gh pr checkout 38

Comment (7VLY6hpcgWXfsyF) [reply README.md:148]: gh pr checkout 38

**Pull request merged**